packages/happychat-connection/src/use-happychat-available.ts

```diff
@@ -8,41 +8,19 @@ type HCAvailability = { available?: boolean; status?: string; env?: 'staging' |
 const key = Date.now();
 
 function getHCAvailabilityAndStatus( dataAuth: HappychatAuth ) {
+	const result: HCAvailability = {};
```

Comment:
One thing about modifying this function is that we will be fetching the authentication still with every request. I am not sure if we need to do that?

Maybe we can have two functions? `authenticateHappychat` and then `useHappychatAvailability`. That way we can set the stale time to something smaller like 60 seconds and check it more frequently without having to authenticate everytime.

Reply:
> is that we will be fetching the authentication still with every request

Why is that?